server/lib/nl/constants.py

```diff
@@ -155,10 +155,75 @@
 
 # TODO: remove this special casing when a better NER model is identified which
 # can always detect these.
-SPECIAL_PLACES: Set[str] = {'palo alto', 'mountain view'}
+SPECIAL_PLACES: Set[str] = {
+    'palo alto', 'mountain view', 'world', 'earth', 'african', 'asian',
+    'european', 'north american', 'south american', 'latin american', 'oceania',
+    'american'
```

Comment:
do we need just "africa" as well?

Reply:
removed 'african', 'asian', 'european', 'north american', 'south american', 'latin american', 'american'